app/models/floating_ip/operations.rb

```diff
@@ -0,0 +1,19 @@
+module FloatingIp::Operations
+  extend ActiveSupport::Concern
+
+  def update_floating_ip(options = {})
+    raw_update
```

Comment:
@gmcculloug also, should I fix `options` argument for `update_floating_ip` method, according to rubocop offence? ^^